src/test/resources/codeflare/koranteng_ray2.yaml

```diff
@@ -0,0 +1,205 @@
+                valueFrom:
+                  fieldRef:
+                    fieldPath: status.podIP
+              - name: RAY_USE_TLS
+                value: '0'
+              - name: RAY_TLS_SERVER_CERT
+                value: /home/ray/workspace/tls/server.crt
+              - name: RAY_TLS_SERVER_KEY
+                value: /home/ray/workspace/tls/server.key
+              - name: RAY_TLS_CA_CERT
+                value: /home/ray/workspace/tls/ca.crt
+              - name: COOKIE_SECRET
+                valueFrom:
+                  secretKeyRef:
+                    key: cookie_secret
+                    name: koranteng-oauth-config
+            image: quay.io/project-codeflare/ray:latest-py39-cu118
+            imagePullPolicy: Always
+            lifecycle:
+              preStop:
+                exec:
+                  command:
+                    - /bin/sh
+                    - -c
+                    - ray stop
+            name: ray-head
+            ports:
+              - containerPort: 6379
+                name: gcs
+              - containerPort: 8265
+                name: dashboard
+              - containerPort: 10001
+                name: client
+            resources:
+              limits:
+                cpu: 2
+                memory: 8G
+                nvidia.com/gpu: 0
+              requests:
+                cpu: 2
+                memory: 8G
+                nvidia.com/gpu: 0
+            volumeMounts:
+              - mountPath: /etc/pki/tls/certs/odh-trusted-ca-bundle.crt
+                name: odh-trusted-ca-cert
+                subPath: odh-trusted-ca-bundle.crt
+              - mountPath: /etc/ssl/certs/odh-trusted-ca-bundle.crt
+                name: odh-trusted-ca-cert
+                subPath: odh-trusted-ca-bundle.crt
+              - mountPath: /etc/pki/tls/certs/odh-ca-bundle.crt
+                name: odh-ca-cert
+                subPath: odh-ca-bundle.crt
+              - mountPath: /etc/ssl/certs/odh-ca-bundle.crt
+                name: odh-ca-cert
+                subPath: odh-ca-bundle.crt
+          - args:
+              - --https-address=:8443
+              - --provider=openshift
+              - --openshift-service-account=koranteng-oauth-proxy
+              - --upstream=http://localhost:8265
+              - --tls-cert=/etc/tls/private/tls.crt
+              - --tls-key=/etc/tls/private/tls.key
+              - --cookie-secret=$(COOKIE_SECRET)
+              - --openshift-delegate-urls={"/":{"resource":"pods","namespace":"test-codeflare","verb":"get"}}
+            image: registry.redhat.io/openshift4/ose-oauth-proxy@sha256:1ea6a01bf3e63cdcf125c6064cbd4a4a270deaf0f157b3eabb78f60556840366
+            name: oauth-proxy
+            ports:
+              - containerPort: 8443
+                name: oauth-proxy
+            resources: {}
+            volumeMounts:
+              - mountPath: /etc/tls/private
+                name: proxy-tls-secret
+                readOnly: true
```

Comment:
oauth-proxy shouldn't be needed from RHOAI 2.9 RC2, it should be added by CodeFlare operator webhook.

Reply:
this is what codeflare-sdk generated for me, so, should I rerun with latest version of codeflare-sdk and it will disappear from the yaml?

Reply:
well, it is not merged yet, once https://github.com/project-codeflare/codeflare-sdk/pull/509 is merged and SDK released then it will be gone

Reply:
just saying it in advance

Reply:
* opened https://github.com/skodjob/odh-e2e/issues/132